clients/admin-ui/src/types/api/models/ConnectionSystemTypeMap.ts

```diff
@@ -15,13 +14,3 @@ export type ConnectionSystemTypeMap = {
   human_readable: string;
   encoded_icon?: string;
 };
-
-export const isConnectionSystemTypeMap = (
```

Comment:
fyi @galvana I moved this from here to `datastore-connections/types.ts` instead, since this file is autogenerated so gets overwritten fairly often